app/src/components/FeesPreview.tsx

```diff
@@ -36,8 +36,9 @@ const FeesPreview: FC<FeesPreviewProps> = ({ loading, fees, hidden }) => {
               <div className="text-turtle-foreground">
                 {feeToHuman(fees)} {fees.token.symbol}
               </div>
-              {/* todo(nuno) */}
-              {/* <div className="text-turtle-level3">${state.fees.inDollars}</div> */}
+              {fees.inDollars > 0 && (
+                <div className="text-turtle-level3">${fees.inDollars.toFixed(10)}</div>
```

Comment:
Why `toFixed(10)` here? 🤔  Can we make it `3` instead?

```suggestion
                <div className="text-turtle-level3">${fees.inDollars.toFixed(3)}</div>
```

Reply:
toFixed(3) will not display the fees (at least on testnet) as they are 0.0000005.

I will keep it like this for now as it is part of a proper issue/improvement from our board that needs to be addressed (see [here]
(https://www.notion.so/velocitylabs/df1914bfea20494c8a0a32af79fbb448?v=aadc09f0be594111b29113e58ee73b42&p=0ace274b2d374dd58b78adfe1cf5360f&pm=s))

Reply:
hm but the price in Dollar should not be a decimal value 🤔

Reply:
What if fees are less than 1$?

Reply:
Sure but it doesn't happen in practise. It's fine if we want to be sure to display enough decimals but the fees involved in the current transfers on mainnet should all be >1$ and if they are not there's probably an error somewhere.